src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PaginationProps = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: ((page:number) => void),
+};
+export const Pagination:React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = getNumbers(1, Math.ceil(total / perPage));
+
+  const canMoveLeft = ():boolean => {
+    return currentPage !== 1;
+  };
```

Comment:
simple variable instead of function can be used
```suggestion
  const canMoveLeft =  currentPage !== 1;
```